src/transformers/models/auto/configuration_auto.py

```diff
@@ -343,6 +345,7 @@
         ("deit", "DeiT"),
         ("detr", "DETR"),
         ("dialogpt", "DialoGPT"),
+        ("dinat", "DiNAT"),
```

Comment:
Those here should be left with proper casing however!

Reply:
Done.